Source/WebCore/platform/cocoa/ParentalControlsContentFilter.mm

```diff
@@ -54,12 +54,12 @@
 
 #if HAVE(WEBCONTENTRESTRICTIONS)
 
-ParentalControlsURLFilter& ParentalControlsContentFilter::impl() const
+Ref<ParentalControlsURLFilter> ParentalControlsContentFilter::impl() const
```

Comment:
Why does this return a `Ref` now? This doesn't seem like a step in the right direction.